### ISSUE TITLE
Too many opened connections 

### ISSUE THREAD

**Issue by dituuu** [crabhi/celery-java#18]
Hello, I'm using the library with a webservice but everytime I publish a message, a connection is opened and never closed. The thing is that after a few hours the RAM gets full because of too many connections and everything crashes.

Is it possible to persist one connections and handle channels?

Thank you,
Diego

ps: sorry for my english

**Comment by crabhi** [comment]
Thanks for the report. How are you using the library? If you use just one client, I think the connection should be reused.

**Comment by dituuu** [comment]
Yes, I use only one client as a static field in my class (like a singleton). But I end up with like 10k connections.
